core/src/server/rpc_module.rs

```diff
@@ -786,9 +746,40 @@ impl SubscriptionSink {
 		self.inner_send(msg).map_err(Into::into)
 	}
 
+	/// Consume the sink by passing a stream to be sent via the sink.
+	pub async fn add_stream<S, T>(mut self, mut stream: S)
```

Comment:
What do you think about calling this `into_stream`? I think "add" implies there could be more than one and that it doesn't quite relay the information about the important changes that this call makes to the sink.

Reply:
Fair; I'd have liked `as_stream` but `as_` is "taken" with different semantics so can't do that.

Of your suggestions I like `from_stream` the best.

Reply:
So, this method consumes a stream, feeding the items into the subscription?

I guess I'd go with something like `consume_stream` or `read_from_stream`. `into_`, `as_`, and `from_` all sortof feel like I should expect some result back from this call to me!

Reply:
I guess we could return a type that impls `Sink/SinkExt` instead here to make it more readable and flexible i.e, to deal with errors and so on.

Reply:
`streamify()`? 
I think `consume_stream` is so-so. Yes, we do consume it, but that's not really the point. Rather we're "hooking up" the stream to the sink and leave it there for the duration of the subscription.
`with_stream`?

Reply:
`pipe`, maybe? we're piping a stream into the subscription

Reply:
`sink.pipe_from_stream`? I quite like `pipe`!

Reply:
I like `pipe_from_stream`, let's settle for that?